<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultado</class>
 <widget class="QDialog" name="resultado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>coffee+tea demo</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RESUMEN DE LA COMPRA</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cliente: </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MostrarNombre">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tipo de caja:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MostrarCaja">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sabor: </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MostrarSabor">
        <property name="maximumSize">
         <size>
          <width>106</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Subtotal: $</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MostrarSubtotal">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Recargo: $</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MostrarRecargo">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>IVA 21%: $</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MostrarIVA">
        <property name="font">
         <font>
          <family>Lovehearts XYZ</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Lovehearts XYZ</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total: $</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="MostrarTotal">
      <property name="font">
       <font>
        <family>Lovehearts XYZ</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string> </string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
